### PR TITLE
JS: Implement in-place aromatisation/kekulisation and avoid undesired exception

### PR DESCRIPTION
**Pull request by ptosco**
Small PR to add in-place aromatisation/kekulisation (similarly to `JSMol::set_new_coords()`) and avoid that an exception is thrown if an attempt is made to aromatise a molecule that was not previously sanitized (e.g., a query mol).

- implemented in-place aromatisation/kekulisation
- avoid that attempting to aromatise a query mol throws an exception due `calcImplicitValence()` not having been called
- add unit tests for the above

### CODE REVIEW

File: Code/MinimalLib/minilib.h [hunk 1]
```diff
@@ -99,7 +99,9 @@ class JSMol {
 
   std::string get_stereo_tags() const;
   std::string get_aromatic_form() const;
+  void set_aromatic_form();
```
Comment (greglandrum) [inline Code/MinimalLib/minilib.h:102]: I think `convert_to_aromatic_form` (and `convert_to_kekule_form`) is a better name.

**Review by greglandrum — COMMENTED** [review]
I'm not a fan of the function names, but this otherwise looks ok to me.

**Comment by ptosco** [comment]
@greglandrum Sure, I'll rename the functions. Please wait before merging.

File: Code/MinimalLib/minilib.h (hunk 1 above)
Comment (ptosco) [reply Code/MinimalLib/minilib.h]: Done.

**Review by greglandrum — APPROVED** [review]
LGTM

**Pull request merged**